docs/get-started/ruby.md

```diff
@@ -11,3 +11,126 @@ parent: Get started
 ## Intializing target project
 
 Before you can compile and run the target project, you will need to initialize it. After initializing the test project, you will need to add references to the [abstraction](https://github.com/microsoft/kiota/tree/main/abstractions/ruby/microsoft_kiota_abstractions), [authentication](https://github.com/microsoft/kiota/tree/main/authentication/ruby/azure), [http](https://github.com/microsoft/kiota/tree/main/http/ruby/nethttp/microsoft_kiota_nethttplibrary), and [serialization](https://github.com/microsoft/kiota/tree/main/serialization/ruby/json/microsoft_kiota_serialization) packages from the GitHub feed.
+
+### Getting access to the packages
+> **Note:** This is a temporary requirement while Kiota is in preview.
+#### Install Bundler
+1. To install bundler, execute the following line in the root directory of your project:
+````shell
+gem install bundler
+````
+2. Create a gemfile named **Gemfile** in the root directory of your project. 
+
+#### Getting your Personal Access Token
+1. Navigate to [New personal access token](https://github.com/settings/tokens/new) and generate a new token. (permissions: `package:read`, `repo`).
+2. Copy the token, you will need it later.
+3. Enable SSO for the Microsoft org on the token if you are a Microsoft employee.
+
+#### Installing the necessary packages
+1. In your project, update your `Gemfile` with the required information. Add these lines to your application's Gemfile:
+
+    ```ruby
+    source 'https://rubygems.org'
+    git_source(:github) { |repo_name| "https://rubygems.pkg.github.com/microsoft" }
+
+    source "https://rubygems.pkg.github.com/microsoft" do
+        gem "microsoft_kiota_abstractions"
+        gem "microsoft_kiota_serialization"
+        gem "microsoft_kiota_authentication_oauth"
+        gem "microsoft_kiota_nethttplibrary"
+    end
+    ```
+    Only the first package, `microsoft_kiota_abstractions`, is required. The other packages provide default implementations that you can choose to     replace with your own implementations if you wish.
+
+2. Use `bundler config` to globally specify the authentication information.
+    Execute this line:
+    ````shell
+    bundle config https://rubygems.pkg.github.com/microsoft/kiota USERNAME:TOKEN     
```

Comment:
the URL here is different from the one in the Gemfile, just want to make sure this is expected.

Reply:
yes, it is!